src/sentry/incidents/endpoints/serializers.py

```diff
@@ -314,6 +315,8 @@ def validate(self, data):
         This includes ensuring there is either 1 or 2 triggers, which each have actions, and have proper thresholds set.
         The critical trigger should both alert and resolve 'after' the warning trigger (whether that means > or < the value depends on threshold type).
         """
+        data["aggregate"] = aggregation_function_translations[data.pop("aggregation")]
```

Comment:
Doesn't this overwrite the aggregate with the aggregation?

Reply:
At the moment we can't actually pass `aggregate` at all, so this is just converting `aggregation` -> `aggregate`. I'm working on the pr now that will allow `aggregate` to be passed.

Reply:
When you say be passed, you mean passed from the api right?

Reply:
Yeah, at the point this pr is in we still only accept `aggregation` and translate it to `aggregate`. I have some follow up prs to this that remove other uses of `aggregation` in the code, which is required before we can accept custom `aggregate` values otherwise we'd end up errorring when we can't translate back from `aggregate` -> `aggregation`.